src/mapgen.cpp

```diff
@@ -3279,26 +3278,32 @@ C..C..C...|hhh|#########\n\
                 rotate(3);
             }
         } else if (tw != 0 || rw != 0 || lw != 0 || bw != 0) { // Sewers!
+
+            const auto is_lab = []( const oter_id &id ) {
```

Comment:
Can you document this? I suspect your trying to stop the lab cutting into sewers?